src/System.Private.ServiceModel/src/System/ServiceModel/Description/XmlSerializerOperationBehavior.cs

```diff
@@ -412,6 +412,12 @@ private void CreateHeaderDescriptionTable(MessageDescription message, MessageInf
                         else if (headersMapping != null)
                         {
                             XmlMemberMapping memberMapping = headersMapping[headerNameIndex++];
+#if FEATURE_NETNATIVE
+                            // We cannot do the checks in Net Native because the mapping
+                            // object is just a fake mapping instance which has a valid key only.
+                            // The checks should have already been done in SG.exe
+                            headerDescriptionTable.Add(header.Name, header.Namespace, header);
```

Comment:
Nit: possibly rephrase comment to not mention SG.exe, which has meaning only to our team.   Maybe just say it was already done during NET Native precompilation.